.travis.yml

```diff
@@ -1,6 +1,15 @@
 language: python
 python:
   - "2.7"
+os: linux
+dist:
+  - xenial
+  - bionic
```

Comment:
trying in 18.04 as well trying to identify any compatibility issue, we might need to create a docker artifact to try to get this up